docs/stargz.md

```diff
@@ -89,3 +89,89 @@ $ nerdctl push example.com/foo:estargz
 NOTE: `--estargz` should be specified in conjunction with `--oci`
 
 Stargz Snapshotter is not needed for building stargz images.
+
+## Tips for image conversion
+
+### Tips 1: Using zstd instead of gzip (a.k.a. zstd:chunked)
+
+You can use zstd compression with lazy pulling support (a.k.a zstd:chunked) instead of gzip.
+
+
+```console
+$ nerdctl build -t example.com/foo .
+$ nerdctl image convert --zstdchunked --oci example.com/foo example.com/foo:zstdchunked
+$ nerdctl push example.com/foo:zstdchunked
+```
+
+### Tips 2: Creating smaller eStargz images
```

Comment:
Tips 2 seems more important than tips 1, so they should be probably swapped

Reply:
Fixed.